src/SupervisionSuccess/constants.js

```diff
@@ -1,2 +1,4 @@
-/* eslint-disable import/prefer-default-export */
-export const CSV_PROCESSING_ERROR = "Error occurred while processing csv file";
+export const ERROR_RESPONSE_NOT_OK = "Could not load data file (response not OK)";
+export const ERROR_NOT_CSV_FETCHED =
+  "Could not load data file (fetched not csv file). Most likely the 404 Page was received.";
+export const ERROR_CHECKPOINTS = "Savings and checkpoints do not match.";
```

Comment:
Let's change this to: "Savings and Checkpoint columns in the params file do not align as expected. Check the documentation in the readme to see the requirements for these columns"